tests/system/aiplatform/e2e_base.py

```diff
@@ -136,7 +136,10 @@ def tear_down_resources(self, shared_state: Dict[str, Any]):
         # Bring all Endpoints to the front of the list
         # Ensures Models are undeployed first before we attempt deletion
         shared_state["resources"].sort(
-            key=lambda r: 1 if isinstance(r, aiplatform.Endpoint) else 2
+            key=lambda r: 1
```

Comment:
I had talked to Morgan before about some ideas to clean this up. This will get uglier as time passes.